Gemfile.lock

```diff
@@ -662,49 +707,49 @@ GEM
       redis-namespace (~> 1.3)
       sinatra (>= 0.9.2)
       vegas (~> 0.1.2)
-    resque-scheduler (4.3.0)
+    resque-scheduler (4.3.1)
       mono_logger (~> 1.0)
-      redis (~> 3.3)
+      redis (>= 3.3, < 5)
       resque (~> 1.26)
       rufus-scheduler (~> 3.2)
     rest-client (2.0.2)
       http-cookie (>= 1.0.2, < 2.0)
       mime-types (>= 1.16, < 4.0)
       netrc (~> 0.8)
     retriable (3.1.1)
-    roo (2.6.0)
+    roo (2.7.1)
```

Comment:
This is the source of the CSV files not being ingested properly. I guess we need to specify exactly roo 2.6.0 in the Gemfile (`=2.6.0` rather than `~> 2.6`).

For reference to the original issue with CSV files not ingesting correctly, see PR #14